terminal.go

```diff
@@ -141,7 +141,8 @@ type LightningTerminal struct {
 	wg         sync.WaitGroup
 	lndErrChan chan error
 
-	lndClient *lndclient.GrpcLndServices
+	lndClient   *lndclient.GrpcLndServices
+	basicClient lnrpc.LightningClient
```

Comment:
Can likely start to retroactively move some of the new macaroon methods in to a new sub-server, but that's outside the scope of this PR.